client/src/components/Home.tsx

```diff
@@ -0,0 +1,21 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import Navbar from "./Navbar";
+function Home() {
+  return (
+    <>
+      <Navbar title="Organize" />
+      <div className="container">
+        <div className="christmas-box">
+          <h1 className="box-text text-center ">
+            {25 - new Date().getDate().valueOf()}
```

Comment:
I was thinking of having the assigned date being up to the user. I'm sure most secret santas are suppose to happen on Christmas day, but what if someone what's to organize the secret-santa event near christmas?